vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/regionutils/RegionFactoryTest.java

```diff
@@ -88,4 +106,29 @@ public void divideRegionInChunks() {
         regions = regionFactory.divideRegionInChunks("1", 3000, 2500);
         assertTrue(regions.size() == 0);
     }
+
+    @Test
+    public void executeQueryForGettingMinAndMaxCoordinatesWhenTheyAreNotIncludedInTheRegionFilter() throws IOException, IllegalOpenCGACredentialsException {
```

Comment:
Still it feels this method could be split in multiple small one. I mean, there is a comment before each fragment explaining what is being tested. It would be more clear to have one method per case so no comments are necessary.